src/main/java/sirius/biz/storage/layer2/URLBuilder.java

```diff
@@ -405,6 +411,29 @@ public boolean isConversionExpected() {
         return isFilled() && !Strings.areEqual(variant, VARIANT_RAW) && Strings.isEmpty(determinePhysicalKey());
     }
 
+    /**
+     * Determines if the conversion of the given variant has finally failed and therefore no further conversion attempts are made.
+     *
+     * @return <tt>true</tt> if the variant is failed, otherwise <tt>false</tt>.
+     */
+    public boolean isConversionFailed() {
```

Comment:
Sounds grammatically strange. I would prefer `hasConversionFailed()`, but that is just an opinion. Also, `is…` is a getter for booleans, and this is no getter.

Reply:
yep, you are right, right above this method is another method with the same naming patter, which is why I setteled on it...but I consider renaming it =)

Reply:
"Des hamm mer immer schon so gemacht" war noch nie ein gutes Argument 😉😅

Reply:
Das ist richtig, wollte nur nix renamend und es irgendwie harmonisch haben